package com.br.alfaEscola.controle.service.exception;

public class ProvaInexistente extends RuntimeException{

}
